xml/System.Net/HttpVersion.xml

```diff
@@ -210,29 +210,6 @@
         <summary>Defines a <see cref="T:System.Version" /> instance for HTTP 2.0.</summary>
         <remarks>To be added.</remarks>
       </Docs>
-    </Member>
-    <Member MemberName="Version30">
-      <MemberSignature Language="C#" Value="public static readonly Version Version30;" />
-      <MemberSignature Language="ILAsm" Value=".field public static initonly class System.Version Version30" />
-      <MemberSignature Language="DocId" Value="F:System.Net.HttpVersion.Version30" />
-      <MemberSignature Language="VB.NET" Value="Public Shared ReadOnly Version30 As Version " />
-      <MemberSignature Language="C++ CLI" Value="public: static initonly Version ^ Version30;" />
-      <MemberSignature Language="F#" Value=" staticval mutable Version30 : Version" Usage="System.Net.HttpVersion.Version30" />
-      <MemberType>Field</MemberType>
-      <AssemblyInfo>
-        <AssemblyName>System.Net.Primitives</AssemblyName>
-        <AssemblyVersion>5.0.0.0</AssemblyVersion>
-      </AssemblyInfo>
-      <AssemblyInfo>
-        <AssemblyName>netstandard</AssemblyName>
-      </AssemblyInfo>
-      <ReturnValue>
-        <ReturnType>System.Version</ReturnType>
-      </ReturnValue>
-      <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
-      </Docs>
-    </Member>
+    </Member>    
```

Comment:
I'm not sure how docs and auto-generating tools will handle this removal.
Maybe it should be left with `To be added.` (which is skipped) until auto-removed. If that's what the tool will do once it sees the property is gone.

Reply:
I believe the tool will recreate the section once it discover the api again. By the way it should also fill comments from `///` in code during first run.

@carlossanlop am I right?

Reply:
Don't remove APIs manually.

If this is an API that has been removed from runtime, then we have to wait for the MS Docs build system to catch up and remove it automatically.

Is this API gone in Preview8 or above? Because we haven't updated MS Docs yet with that preview.